gtk/MessageLogWindow.cc

```diff
@@ -455,14 +473,11 @@ MessageLogWindow::Impl::Impl(
     , refresh_tag_(Glib::signal_timeout().connect_seconds(
           sigc::mem_fun(*this, &Impl::onRefresh),
           SECONDARY_WINDOW_REFRESH_INTERVAL_SECONDS))
-    , level_names_{ {
-          { TR_LOG_CRITICAL, C_("Logging level", "Critical") },
-          { TR_LOG_ERROR, C_("Logging level", "Error") },
-          { TR_LOG_WARN, C_("Logging level", "Warning") },
-          { TR_LOG_INFO, C_("Logging level", "Information") },
-          { TR_LOG_DEBUG, C_("Logging level", "Debug") },
-          { TR_LOG_TRACE, C_("Logging level", "Trace") },
-      } }
+    , level_names_{ { { TR_LOG_CRITICAL, C_("Logging level", "Critical") },
+                      { TR_LOG_ERROR, C_("Logging level", "Error") },
+                      { TR_LOG_WARN, C_("Logging level", "Warning") },
+                      { TR_LOG_INFO, C_("Logging level", "Information") },
+                      { TR_LOG_DEBUG, C_("Logging level", "Debug") } } }
```

Comment:
I usually add trailing comma to avoid such extra and uneven indentation.

Reply:
Trailing comma added in 7dc51830a91252b0f06a80c51dd78687d58ff7d6

Reply:
Slightly misplaced, should be after the element, not the whole value ;)

Reply:
In my defense, I saw the error before reading your comment -- but I appreciate the review either way :)